scripts/parser-tests/flow/allowlist.txt

```diff
@@ -1,7 +1,8 @@
-# 26 invalid programs did not produce a parsing error
+# 27 invalid programs did not produce a parsing error
 
 JSX/invalid_unpaired_gt.js
 JSX/invalid_unpaired_rcurly.js
+JSX/jsx_type_args_types_disabled.js
```

Comment:
In this test, `<Component<T> />` is parsed with the flow parser option `{ types: false }`, but we don't have a mirroring parser option.